src/sentry/tsdb/redis.py

```diff
@@ -339,7 +339,7 @@ def get_range(
         self.validate_arguments([model], [environment_id])
 
         rollup, series = self.get_optimal_rollup_series(start, end, rollup)
-        _series: list[datetime] = [to_datetime(item) for item in series]
+        _series = [to_datetime(item) for item in series]
```

Comment:
`series` here is a sequence of `int`s -- so below the `.timestamp()` will return whole number floats that we can safely turn back into ints without truncation